react/components/SearchResultContainer.js

```diff
@@ -128,9 +129,9 @@ export default class SearchResultContainer extends Component {
 
     this._fetchMoreLocked = true
 
-    const { maxItemsPerPage, searchQuery: { products } } = this.props
+    const { maxItemsPerPage, searchQuery: { products, recordsFiltered } } = this.props
 
-    const to = maxItemsPerPage + products.length - 1
+    const to = Math.min(+maxItemsPerPage + products.length, recordsFiltered) - 1
```

Comment:
You can use ramda functions to this, like composing `subtract`  with `min` and `sum`. What you think?

Reply:
```js
const to = subtract(min(sum(maxItemsPerPage, products.length), recordsFiltered), 1)
```
It would be something like that, right?
I think that using the operators is more idiomatic.

Reply:
I prefer this solution anyway, but i will pass it and can be discussed forwardly